modules/statics/README.md

```diff
@@ -33,15 +33,15 @@ console.log(btc.decimalPlaces);
 
 ### Get the contract address for the OmiseGo ERC20 Token
 
-#### Javascript
+#### JavaScript
 ```js
 const { coins } = require('@bitgo/statics');
 
 const omg = coins.get('omg');
 console.log(omg.contractAddress);
 ```
 
-#### Typescript
```

Comment:
actually missed this. was merged but think this is a typo. same with other places it got changed back to `typescript`